wayshot/src/clap.rs

```diff
@@ -57,4 +57,12 @@ pub fn set_flags() -> Command {
                 .conflicts_with("slurp")
                 .help("Choose a particular display to screenshot"),
         )
+        .arg(
+            arg!(--dialog)
```

Comment:
would `--choose-output` be a better naming scheme?

Reply:
but clap do not allow name have '-', emm